.github/workflows/alphabetic-userguide.py

```diff
@@ -0,0 +1,42 @@
+from bs4 import BeautifulSoup
+import yaml 
+import sys
+
+en_yaml = "_i18n/en.yml"
+index = "resources/user-guides/index.md"
+
+with open(en_yaml, 'r') as stream:
+    data_loaded = yaml.safe_load(stream)
+with open(index, "r") as f:
+    soup = BeautifulSoup(f, "html.parser")
+
+divs = soup.find_all("div", {"class": "col"})
+data = {}
+for div in divs:
+    category = str(div.find("h2"))
+    category = category.split(".")[1].split(" ")[0]
+    data[category] = []
+    p = div.find_all("p")
+    for lol in p:
```

Comment:
As this is something we might merge to the repo it would probably be better to use a proper variable name here :D

Reply:
Oops, thank you, my mind does go blank sometimes, adjusted it now, should be ok :+1: